dashboard/src/components/QueryResult.tsx

```diff
@@ -1,14 +1,27 @@
 import EmptyState from './EmptyState';
-import { Label, Spinner, Table, VBox } from './common';
+import { Label, Spinner, AWSSpinner, Table, VBox } from './common';
 import { useAppContext } from './providers/AppProvider';
 
 export default function QueryResult() {
   const { editorSelectedTab, editorTabs } = useAppContext();
   const queryRes = editorTabs[editorSelectedTab]?.queryRes;
+  const query = editorTabs[editorSelectedTab]?.content;
+  const longRunningRpc = query.includes('iasql_commit()') || query.includes('iasql_install(');
 
   // TODO: show all statements with its respective query and not just the last one
   return editorTabs?.[editorSelectedTab]?.isRunning ? (
-    <Spinner />
+    longRunningRpc ? (
+      <VBox>
+        <p className='pt-20 px-20 font-bold'>
+          We are importing the entire state of the infrastructure from the cloud account and due to default
```

Comment:
maybe the `We are importing the entire state...` does not align 100% with `iasql_commit`?

Reply:
technically we do a sync on commit, but yes. perhaps `reading` or `parsing`?

Reply:
What about: `Cloud operations can take several minutes due to default cloud SDK rate limiters depending on how many resources are in the account.  Please be patient.`